src/encoded/types/functional_characterization_experiment.py

```diff
@@ -339,3 +432,4 @@ def biosample_summary(self, request, replicates=None, elements_references=None):
 
         if len(dictionaries_of_phrases) > 0:
             return construct_biosample_summary(dictionaries_of_phrases, sentence_parts)
+                        
```

Comment:
No newline. I am surprised the linter accepted that.

Reply:
fixed